inst/shell_scripts/iauth.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/sh
 
 # get token
-SECRETS=$(echo -n "$1:$2" | base64)
-echo $(curl -X POST -H "Authorization: Basic ${SECRETS}" "$3/auth")
+SECRETS=$(echo -n "rods:rods" | base64)
+TOKEN=$(curl -X POST -H "Authorization: Bearer ${SECRETS}" "$3/authenticate")
+
+curl http://localhost:9001/irods-http-api/0.1.0/data-objects \
+    -H "Authorization: Bearer ${SECRETS}" \
+    --data-urlencode 'op=parallel-write-init' \
+    --data-urlencode 'lpath=<string>' \
+    --data-urlencode 'stream-count=<integer>' \
+    --data-urlencode 'truncate=<integer>' \
+    --data-urlencode 'append=<integer>' \
+    --data-urlencode 'ticket=<string>'
```

Comment:
i'm not sure what this is doing to do... there are no values for lpath, and others...

Reply:
I need to remove some files here. They where used for initial development and testing (comparing curl with libcurl + R output).

Reply:
Only some scripts are actually used in the package in relation with `use_irods_demo()`.

Reply:
I have removed those scripts now